### PR TITLE
[Feat] Add APC ATS entity

### PR DESCRIPTION
**Pull request by PortableProgrammer**
### Relevant information

Adds a new entity for APC Automated Transfer Switches. Pairs with [kentik/snmp-profiles #733](https://github.com/kentik/snmp-profiles/pull/733). As with the SNMP profile, tested on AP7752, which doesn't support the full MIB.

### Checklist

* [x] I've read the guidelines and understand the acceptance criteria.
* [x] The value of the attribute marked as `identifier` will be unique and valid. 
* [x] I've confirmed that my entity type wasn't already defined. If it is I'm providing an explanation above.

### CODE REVIEW

**Comment by lujop** [comment]
Also take a look at validator output that is complaining about several issues in the dashboard definition.
For example you need to add `"accountId": 0` in the `nrqlQueries` objects. 
You can check another existing dashboard definition if it helps your.
 If you have trouble understanding any of the validator errors, please let us know by replying here.

File: entity-types/ext-ats/golden_metrics.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+outputLoadCapacity:
+  title: Output load (amps)
+  unit: COUNT
```
[the remark this answers is not in the document]
Comment (PortableProgrammer) [reply entity-types/ext-ats/golden_metrics.yml:3]: @lujop The list given in the documentation doesn't include 'AMPS', which is what this unit _should_ be. I used the `ext-pdu` entity as a template, and it uses `COUNT` as the unit for this same type of metric.

File: entity-types/ext-ats/kentik-dashboard.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+			  "yAxisRight": {
+				"zero": true
+			  }
+			}
+		  },
+		  {
+			"title": "Overall System Status",
+			"layout": {
+			  "column": 3,
+			  "row": 4,
```
[the remark this answers is not in the document]
Comment (PortableProgrammer) [reply entity-types/ext-ats/kentik-dashboard.json:117]: Correct, the first widget is 8 rows high and 2 columns wide, so rows 2-8 will start in column 3. I pulled this JSON directly from a test dashboard I put together using live data from this entity.

**Comment by lujop** [comment]
The validator is failing because the dashboard definition is not sanitized.
You need to sanitize your dashboard definiton, look at https://github.com/newrelic/entity-definitions/blob/6d8e67cb43fc22e6f1408d9b1ec25cb4eae6366b/validator/README.md?plain=1#L26

**Comment by otaviocarvalho** [comment]
Closing due to inactivity, please feel free to re-open anytime.

**Pull request closed without merging**